src/js/jsx/sections/transform/Size.jsx

```diff
@@ -78,6 +139,64 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Saves the current width of selected layers on scrub start
+         *
+         * @private
+         */
+        _handleWScrubBegin: function () {
+            this._initialScrubW = collection.uniformValue(this.state.widths);
+        },
+
+        /**
+         * Update the current width of the selected layers based on change from initial scrub position
+         *
+         * @private
+         * @param {number} deltaX
+         */
+        _handleWScrub: function (deltaX) {
+            if (_.isNumber(this._initialScrubW)) {
+                var document = this.props.document,
+                    uiStore = this.getFlux().store("ui"),
+                    resolvedDelta = uiStore.zoomCanvasToWindow(deltaX),
+                    newWidth = Math.max(this._initialScrubW + resolvedDelta, this.refs.width.props.min);
+
+                if (newWidth > 0) {
+                    this.getFlux().actions.transform.setSizeThrottled(document, document.layers.selected,
+                        { w: newWidth }, this.props.referencePoint, { coalesce: true });
+                }
+            }
+        },
+
+        /**
+         * Saves the current height of selected layers on scrub start
+         *
+         * @private
+         */
+        _handleHScrubBegin: function () {
+            this._initialScrubH = collection.uniformValue(this.state.heights);
+        },
+
+        /**
+         * Update the current height of the selected layers based on change from initial scrub position
+         *
+         * @private
+         * @param {number} deltaX
+         */
+        _handleHScrub: function (deltaX) {
+            if (_.isNumber(this._initialScrubH)) {
+                var document = this.props.document,
+                    uiStore = this.getFlux().store("ui"),
+                    resolvedDelta = uiStore.zoomCanvasToWindow(deltaX),
+                    newHeight = Math.max(this._initialScrubH + resolvedDelta, this.refs.height.props.min);
+
+                if (newHeight > 0) {
+                    this.getFlux().actions.transform.setSizeThrottled(document, document.layers.selected,
+                        { h: newHeight }, this.props.referencePoint, { coalesce: true });
```

Comment:
The initial `setSize` needs to be non-coalesced.... I don't see that happening here, unless I'm missing it